src/Core/Settings.h

```diff
@@ -526,6 +526,8 @@ class IColumn;
     M(Int64, max_partitions_to_read, -1, "Limit the max number of partitions that can be accessed in one query. <= 0 means unlimited.", 0) \
     M(Bool, check_query_single_value_result, true, "Return check query result as single 1/0 value", 0) \
     M(Bool, allow_drop_detached, false, "Allow ALTER TABLE ... DROP DETACHED PART[ITION] ... queries", 0) \
+    M(UInt64, max_table_size_to_drop, 0, "Only available in ClickHouse Cloud", 0) \
```

Comment:
The description should be full. It should be identical in the public and the private repositories.

Reply:
not sure if the plan is to keep this feature to the internal fork but I don't think it should be in the open-source version if there is no such feature

Reply:
This will be available in open-source.